### ISSUE TITLE
Disconnexion while reading historicals values

### ISSUE THREAD

**Issue by Sebwap** [vrachieru/xiaomi-flower-care-api#4]
Hello,
I've tried to use your readme file informations to write a javascript page allowing to view data from my flower care.

Everything is OK, except that I often face disconnexion while reading historical values, about every 30 to 40 values.

Did you face this issue to ? Is there any workaround to manage this ?

Thanks a lot for your clear documentation about xiaomi protocol !

**Comment by mtopolewski** [comment]
Yeah, I had the same issue on my implementation with C#. Unfortunately, I didn't found any soluton :/
